six/modules/c++/six.sidd/include/six/sidd/DerivedDataBuilder.h

```diff
@@ -80,7 +80,7 @@ class DerivedDataBuilder
      *  \param regionType What kind of region type scheme does the product use
      *  \return Reference to self
      */
-    virtual DerivedDataBuilder& addGeographicAndTargetOld(RegionType regionType);
+    virtual DerivedDataBuilder& addGeographicAndTarget(RegionType regionType);
```

Comment:
do you currently use:
1. addGeographicAndTarget & addGeoData for SIDD 2.0 and just addGeographicAndTarget for 1.0
2. addGeoData for SIDD 2.0 and addGeographicAndTarget for 1.0

based on the comments looks like the second. but i thought based on the code needed to use both (1.)

Reply:
Nope the main change of this PR was to make it 2.
Should have mentioned that to you lol

Reply:
👍 thanks, this looks correct then.